### PR TITLE
Update change log for 7.8 release

### PR DESCRIPTION
**Pull request by hjoliver**
@cylc/core - anyone want to take a quick look at this, to see if I missed anything important?

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -5,10 +5,132 @@ all changes see the [closed
...
+## __cylc-7.8.0 (2018-11-27)__
+
+Minor release with over 120 issues closed. Significant issues include:
+
+### Enhancements
+
+[#2809](https://github.com/cylc/cylc/pull/2809) - __auto-migration__ - suite
+server programs can now be told (via global config) to self-migrate to
+another available host, e.g. for server maintenance. (The pool of cylc hosts 
+should see a shared filestem).
```
Comment (kinow) [inline CHANGES.md]: Filesystem typo

File: CHANGES.md [hunk 2]
```diff
@@ -5,10 +5,132 @@ all changes see the [closed
...
+
+[#2614](https://github.com/cylc/cylc/pull/2614) and 
+[#2821](https://github.com/cylc/cylc/pull/2821) - __web-based job log viewer__ -
+ `cylc review` (migration of "Rose Bush" from the Rose project). 
+
+[#2693](https://github.com/cylc/cylc/pull/2693) - __auto host selection__ -
+`cylc run` and `cylc restart` can now select the best available host based on
+several metrics, on which to launch the suite server program. See `cylc
+get-host-metrics` and global config documentation. (The pool of cylc hosts
+should see a shared filestem).
```
Comment (kinow) [inline CHANGES.md]: Filesystem typo

File: CHANGES.md [hunk 3]
```diff
@@ -5,10 +5,132 @@ all changes see the [closed
...
+- Jinja2Tests and Jinja2Globals, for custom "is" tests and global variables
+  (c.f. our existing Jinja2Filters for custom filters).
+
+[#2682](https://github.com/cylc/cylc/pull/2682) - new built-in
+Jinja2 filter to convert ISO8601 date-time durations to
+decimal seconds or hours.
+
+[#2842](https://github.com/cylc/cylc/pull/2842) - `cylc gui` and
+`cylc graph` - better integration with system desktop themes, including dark
+themes; and other minor graph visualization improvments.
```
Comment (kinow) [inline CHANGES.md]: Improvements typo

File: CHANGES.md [hunk 4]
```diff
@@ -5,10 +5,132 @@ all changes see the [closed
...
+[#2809](https://github.com/cylc/cylc/pull/2809) - __auto-migration__ - suite
+server programs can now be told (via global config) to self-migrate to
+another available host, e.g. for server maintenance. (The pool of cylc hosts 
+should see a shared filestem).
+
+[#2614](https://github.com/cylc/cylc/pull/2614) and 
+[#2821](https://github.com/cylc/cylc/pull/2821) - __web-based job log viewer__ -
+ `cylc review` (migration of "Rose Bush" from the Rose project). 
+
+[#2693](https://github.com/cylc/cylc/pull/2693) - __auto host selection__ -
```
Comment (matthewrmshin) [inline CHANGES.md]: You may want to bundle this with #2809 above.

File: CHANGES.md [hunk 5]
```diff
@@ -5,10 +5,132 @@ all changes see the [closed
...
+[#2807](https://github.com/cylc/cylc/pull/2807) - task output events (event
+handlers can now be triggered when a task reports a registered output message).
+
+[#2868](https://github.com/cylc/cylc/pull/2868) - a new task
+runtime config item `exit-script`, for scripting to be executed at the
+last moment after successful job completion. (Companion of `err-script`).
+
+[#2781](https://github.com/cylc/cylc/pull/2781) and
+[#2854](https://github.com/cylc/cylc/pull/2854) - improved suite
+server program logging.
```
Comment (matthewrmshin) [inline CHANGES.md]: May want to mention that `log/suite/err` is gone.

File: CHANGES.md (hunk 1 above)
Comment (hjoliver) [reply CHANGES.md]: Oops!

File: CHANGES.md (hunk 2 above)
Comment (hjoliver) [reply CHANGES.md]: (oops - copy-n-paste from previous oops!)

File: CHANGES.md [hunk 6]
```diff
@@ -5,10 +5,132 @@ all changes see the [closed
...
+https://github.com/metomi/rose/pull/2252)
+
+[#2812](https://github.com/cylc/cylc/pull/2812) - `cylc gscan`:
+show application menu bar by default.
+
+[#2768](https://github.com/cylc/cylc/pull/2768) - `cylc gscan`: display Cylc
+version of running suites.
+
+[#2786](https://github.com/cylc/cylc/pull/2786) - make task try number
+available to event handlers (as for task job submit number) .
```
Comment (matthewrmshin) [inline CHANGES.md]: Remove space between `)` and `.`?

**Comment by hjoliver** [comment]
I don't normally mention isodatetime updates, treating is as an internal library that users hopefully don't need to know about (happy to change if others prefer though).

**Comment by matthewrmshin** [comment]
(No preference on isodatetime - it is not really relevant to users - so happy for it to be omitted.)

File: CHANGES.md (hunk 4 above)
Comment (hjoliver) [reply CHANGES.md]: Done.

File: CHANGES.md (hunk 6 above)
Comment (hjoliver) [reply CHANGES.md]: Done.

File: CHANGES.md (hunk 5 above)
Comment (hjoliver) [reply CHANGES.md]: Done.

**Comment by hjoliver** [comment]
Thanks for the feedback @kinow and @matthewrmshin - I'm going ahead with this. "Time is of the essence" ... and I don't want to have to eat my :tophat:

**Pull request merged**